pom.xml

```diff
@@ -1545,6 +1557,7 @@
 		<module>hibernate-dialect</module>
 		<module>eclipselink-platform</module>
 		<module>jboss-integration</module>
+		<module>swagger-integration</module>
```

Comment:
There is no need to for separate "Swagger-integration" module, move this into test-integration-common